Exception caught: local variable 'can_id' referenced before assignment
Exception caught: local variable 'can_id' referenced before assignment
Exception caught in background thread.  Unable to handle.
['azure.iot.device.common.transport_exceptions.ConnectionDroppedError: Unexpected disconnection\n']
Exception caught: local variable 'can_id' referenced before assignment
Exception caught: local variable 'can_id' referenced before assignment
Exception caught: local variable 'can_id' referenced before assignment
Exception caught in background thread.  Unable to handle.
